src/main/java/com/github/onsdigital/perkin/SurveyListener.java

```diff
@@ -141,7 +149,11 @@ public String test() throws IOException {
 
     @Override
     public void handleRecovery(Recoverable recoverable) {
-        log.warn("QUEUE|CONNECTION|END|queue connection was recovered");
+        log.warn("QUEUE|CONNECTION|END|queue connection was recovered: {}", recoverable);
```

Comment:
ah, ok, that object doesn't have a toString() implementation so it's defaulted to Object.toString().
I'll check if there's anything useful in the Recoverable object to log...